src/labels/CustomLabels.labels

```diff
@@ -7264,6 +7264,54 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item Train Users in Get Started</shortDescription>
         <value>Train Your users</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistItemWhatIsNPCloudLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for What is NonProfit Cloud checklist item in Welcome section</shortDescription>
+        <value>Learn more about the Nonprofit Cloud in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemWhatIsNPCloudTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for checklist item What is NonProfit Cloud in Welcome section</shortDescription>
+        <value>What is NonProfit Cloud?</value>
```

Comment:
Please change NonProfit to Nonprofit in both the value and the shortDescription.